views/components/flexible-content-next/macro.njk

```diff
@@ -63,10 +63,18 @@
                         ) }}
                     {% elseif part.type === 'factRiver' %}
                         {{ factRiver(part.content) }}
-                    {% elseif part.type === 'tableOfContents' and flexibleContent %}
+                    {% elseif part.type === 'tableOfContents' %}
```

Comment:
Remove this because we are inside a loop for `flexibleContent` so this is always true.